### DISCUSSION TITLE
"unique" validation doesn't work with model properties

### DISCUSSION THREAD

**Discussion by elfeffe** [livewire/livewire#4304] [7. Bugs (archived)] [answered]
If we use model.property, like partner.email, the unique validation fails.

SQLSTATE[42S22]: Column not found: 1054 Unknown column 'partner.email' in 'where clause' (SQL: select count(*) as aggregate from `partners` where `partner`.`email` = test@gmail.com)

Because the column name is not partner.email

**Comment by jonneroelofs** [comment]
Can you show your validation rule?

**Comment by joshhanley** [comment, marked as answer]
@elfeffe that's correct, because the property in Livewire that is being validated is `partner.email` but you want to validate against the `email` column, so you need to specify the column name like `unique:users,email`.

Laravel tries to be smart and use the name for you, hence why it's trying to find a column `partner.email`, but if they don't match you just need to manually specify.

Hope this helps!
